docker-compose.yml

```diff
@@ -0,0 +1,251 @@
+---
+    version: '2.1'
+    
+    services:
+      controller:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        command: -c "/usr/bin/tail -f /dev/null"
+        entrypoint: "/bin/sh"
+        depends_on:
+          - "upstream-tidb"
+          - "downstream-tidb"
+        restart: on-failure
+
+      capturer0:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        entrypoint: "/cdc server"
+        command: 
+          - --pd-endpoints=http://upstream-pd:2379
+          - --log-file=/logs/capturer0.log
+        depends_on:
+          - "upstream-tidb"
+          - "downstream-tidb"
+        restart: on-failure
+
+      capturer1:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        entrypoint: "/cdc server"
+        command: 
+          - --pd-endpoints=http://upstream-pd:2379
+          - --log-file=/logs/capturer1.log
+        depends_on:
+          - "upstream-tidb"
+          - "downstream-tidb"
+        restart: on-failure
+
+      capturer2:
+        image: ticdc:latest
+        build:
+          context: .
+          dockerfile: ./docker/Dockerfile
+        volumes:
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        entrypoint: "/cdc server"
+        command: 
+          - --pd-endpoints=http://upstream-pd:2379
+          - --log-file=/logs/capturer2.log
+        depends_on:
+          - "upstream-tidb"
+          - "downstream-tidb"
+        restart: on-failure
+    
+      upstream-pd:
+        image: pingcap/pd:latest
+        ports:
+          - "2379:2379"
+        volumes:
+          - ./docker/config/pd.toml:/pd.toml:ro
+          - ./docker/data:/data
+          - ./docker/logs:/logs
+        command:
+          - --name=upstream-pd
+          - --client-urls=http://0.0.0.0:2379
+          - --peer-urls=http://0.0.0.0:2380
+          - --advertise-client-urls=http://upstream-pd:2379
+          - --advertise-peer-urls=http://upstream-pd:2380
+          - --initial-cluster=upstream-pd=http://upstream-pd:2380
+          - --data-dir=/data/upstream-pd
+          - --config=/pd.toml
+          - --log-file=/logs/upstream-pd.log
+        restart: on-failure
+    
+      upstream-tikv0:
+        image: pingcap/tikv:latest
```

Comment:
Found the `lastest` tag for `tidb/tikv/pd` is not always the `lastet` version, maybe it is the latest release version. For example at right now I will get the `v3.0.11` from these images. What about using `pingcap/tikv:nightly`, which is always the latest master version